File: ClientApp/src/Domain/Tasks/Models/Enums/PriorityType.ts
export enum PriorityType {
    Low = 1,
    Medium = 2,
    High = 3
}

export function getPriorityTypeDisplayName(type: PriorityType){
    switch(type){
        case PriorityType.Low:
            return 'Низкий'
        case PriorityType.Medium:
            return 'Средний'
        case PriorityType.High:
            return 'Высокий'
    }
}

File: Tasks/Domain/Tasks/Enums/PriorityType.cs
namespace Tools.Enums;

public enum PriorityType
{
    Low = 1,
    Medium = 2,
    High = 3
}

public class PriorityTypeExtensions
{
    public static Boolean IsExist<T>(T type)
    {
        if (type is null) return false;

        if (Enum.IsDefined(typeof(PriorityType), type)) return true;

        return false;
    }
}
